src/main/java/game/pojo/Country.java

```diff
@@ -11,19 +11,21 @@ public class Country {
     List<Country> neighbours;
     Player player;
     int armyCount;
+    List<Integer> borderIds;
```

Comment:
This need not be present as we already have `List<Country> neighbours;`. So you can remove all the changes made in Country.java and just use neighbors in your code in Showmap.java.

Reply:
Done